web-app/src/app/app/jobs/[agentId]/@right/page.tsx

```diff
@@ -0,0 +1,37 @@
+import { notFound } from "next/navigation";
+
+import CreateJobSection from "@/app/jobs/[agentId]/components/create-job-section";
+import {
+  getAgentById,
+  getAgentInputSchema,
+} from "@/lib/db/services/agent.service";
+import { calculateAgentCreditCost } from "@/lib/db/services/credit.service";
+
+interface JobPageParams {
+  agentId: string;
+}
+
+export default async function RightPage({
+  params,
+}: {
+  params: Promise<JobPageParams>;
+}) {
+  const { agentId } = await params;
+
+  const agent = await getAgentById(agentId);
+  if (!agent) {
+    console.log("agent not found in right page");
+    notFound();
+  }
+
+  const agentPrice = await calculateAgentCreditCost(agent);
+  const inputSchema = await getAgentInputSchema(agentId);
```

Comment:
Same as with the other. A loading.tsx would be nice for the user if it takes time.

Reply:
resolved in https://github.com/masumi-network/sokosumi/pull/248/commits/b8729c190c2f55cc1cd21eaadb87a1c77bec8b14